src/Makefile.am

```diff
@@ -9,15 +9,28 @@ CXXFLAGS = -Wall -g -O3 -std=c++11
 LDFLAGS = -lpthread
 
 if DIST_WS
-HUPCPP_FLAGS_1 = -DDIST_WS
+HC_FLAGS_1 = -DDIST_WS
```

Comment:
I think this would confuse reading the code. HUPCPP_* naming convention make it easier to read. Next when we will have HCOpenSHMM, then there will be another flag as HOSHMM_*, making easier to understand without coming back to this file to see what it means.